WordOfTheDay.Repository/WordDA.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using WordOfTheDay.Entities;
+using System.Threading;
+
+namespace Data_access
```

Comment:
Change namespaces in all files according to ProjectName. So e.g. for the Domain project, it should be WordOfTheDay.Domain